WordPress/src/main/res/layout/wpwebview_activity.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/webview_wrapper"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include
+        layout="@layout/toolbar"
+        android:id="@+id/toolbar" />
+
+    <include
+        layout="@layout/progress_layout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@+id/toolbar"
+        tools:visibility="gone"/>
+
+    <WebView
+        android:id="@+id/webView"
+        android:layout_below="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <org.wordpress.android.ui.ActionableEmptyView
+        android:id="@+id/actionable_empty_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/background_default"
+        android:layout_below="@+id/toolbar"
+        android:visibility="gone"
+        app:aevButton="@string/retry"
+        app:aevImage="@drawable/img_illustration_empty_results_216dp"
+        app:aevTitle="@string/error_generic_network"
+        tools:visibility="visible">
+    </org.wordpress.android.ui.ActionableEmptyView>
+
+    <ProgressBar
+        android:id="@+id/progress_bar"
```

Comment:
Maybe I missed something, but are we using `progress_bar` anywhere?

Reply:
This shows up at the top of the Webview (1st gif at the very top of this PR), as the page loads. It may be a little redundant. For some reason when I run the code on 3 of the physical devices that I have around I never get that in "between-state" from the error page to the fully loaded page, but when I try to run it on the emulator I get that every time, so I'm not sure if it's an emulator thing. I was thinking about playing around and possibly adding a bit of delay before displaying the loaded page. Thoughts?